src/coreclr/vm/ecalllist.h

```diff
@@ -600,6 +597,8 @@ FCFuncEnd()
 
 FCFuncStart(gMethodTableFuncs)
     FCFuncElement("GetNumInstanceFieldBytes", MethodTableNative::GetNumInstanceFieldBytes)
+    FCFuncElement("IsEnum", MethodTableNative::IsEnum)
+    FCFuncElement("GetEEClassCorElementType", MethodTableNative::GetEEClassCorElementType)
```

Comment:
Could these three be  QCalls, next to the one added in https://github.com/dotnet/runtime/commit/b6c30d1252b38e65b2808d28cc50ce12dfef60c5?

Reply:
The overhead of QCall could be significant here.